frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -316,36 +324,31 @@ export const getInternalImageInitialValues = (editAppResource: K8sResourceKind)
     'metadata.labels["app.openshift.io/runtime-version"]',
     '',
   );
-  const deployImageInitialValues = {
-    searchTerm: '',
+  return {
+    ...deployImageInitialValues,
     registry: 'internal',
```

Comment:
We could use the enum here RegistryType.Internal.

Reply:
Done